extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-mysql/src/main/java/org/apache/guacamole/auth/mysql/MySQLAuthenticationProviderModule.java

```diff
@@ -95,7 +95,7 @@ public MySQLAuthenticationProviderModule(MySQLEnvironment environment)
         File trustStore = environment.getMySQLSSLTrustStore();
         if (trustStore != null)
             driverProperties.setProperty("trustCertificateKeyStoreUrl",
-                    trustStore.getAbsolutePath());
+                    String.valueOf(trustStore.toURI()));
```

Comment:
Instead of `String.valueOf()` it seems like it would be better to use the `URI` class's `toString()` method, here.  It's possible that's already what `String.valueOf()` does under the hood, but seems like a more direct reference to it would be better.